src/EnergyPlus/CMakeLists.txt

```diff
@@ -933,6 +936,12 @@ if( BUILD_TESTING )
     add_test(NAME "API.PluginTesterTester"
             COMMAND "${PYTHON_EXECUTABLE}" "${API_TEST_TARGET_DIR}/plugin_tester.py" "${PROJECT_SOURCE_DIR}/testfiles/PythonPluginCustomTrendVariable.py"
             WORKING_DIRECTORY "${API_TEST_TARGET_DIR}")
+
+    configure_file( "${PROJECT_SOURCE_DIR}/tst/EnergyPlus/api/test_DataTransferTypes.py" "${API_TEST_TARGET_DIR}/test_DataTransferTypes.py" COPYONLY)
+    add_test(NAME "API.test_DataTransferTypes"
+            COMMAND "${PYTHON_EXECUTABLE}" -m pytest "${API_TEST_TARGET_DIR}/test_DataTransferTypes.py"
+            WORKING_DIRECTORY "${API_TEST_TARGET_DIR}")
+
```

Comment:
Register a new ctest that will call `pytest`

Reply:
I like this but I would like to see it just use `unittest` instead of pytest.  I don't anticipate a huge set of Python unit test files that we have to manage as it would be for an actual Python package, so just using `unittest` should be sufficient.

Reply:
Done in https://github.com/NREL/EnergyPlus/pull/8181/commits/f6829d88bee8642cbe780922d354ddd26d19d205